resources/acm_certificates.go

```diff
@@ -0,0 +1,253 @@
+			},
+			{
+				Name:        "issued_at",
+				Description: "The time at which the certificate was issued",
+				Type:        schema.TypeTimestamp,
+			},
+			{
+				Name:        "issuer",
+				Description: "The name of the certificate authority that issued and signed the certificate.",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "key_algorithm",
+				Description: "The algorithm that was used to generate the public-private key pair.",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "key_usages",
+				Description: "A list of Key Usage X.509 v3 extension objects. Each object is a string value that identifies the purpose of the public key contained in the certificate.",
+				Type:        schema.TypeStringArray,
+				Resolver:    resolveACMCertificateKeyUsages,
+			},
+			{
+				Name:        "not_after",
+				Description: "The time after which the certificate is not valid.",
+				Type:        schema.TypeTimestamp,
+			},
+			{
+				Name:        "not_before",
+				Description: "The time before which the certificate is not valid.",
+				Type:        schema.TypeTimestamp,
+			},
+			{
+				Name:        "certificate_transparency_logging_preference",
+				Description: "You can opt out of certificate transparency logging by specifying the DISABLED option",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("Options.CertificateTransparencyLoggingPreference"),
+			},
+			{
+				Name:        "renewal_eligibility",
+				Description: "Specifies whether the certificate is eligible for renewal",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "renewal_summary_domain_validation_options",
+				Description: "Contains information about the validation of each domain name in the certificate, as it pertains to ACM's managed renewal.",
+				Type:        schema.TypeJSON,
+				Resolver: resolveACMCertificateJSONField(func(cd *types.CertificateDetail) interface{} {
+					if cd.RenewalSummary == nil {
+						return nil
+					}
+					return cd.RenewalSummary.DomainValidationOptions
+				}),
+			},
+			{
+				Name:        "renewal_summary_renewal_status",
+				Description: "The status of ACM's managed renewal (https://docs.aws.amazon.com/acm/latest/userguide/acm-renewal.html) of the certificate.  This member is required.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("RenewalSummary.RenewalStatus"),
+			},
+			{
+				Name:        "renewal_summary_updated_at",
+				Description: "The time at which the renewal summary was last updated.  This member is required.",
+				Type:        schema.TypeTimestamp,
+				Resolver:    schema.PathResolver("RenewalSummary.UpdatedAt"),
+			},
+			{
+				Name:        "renewal_summary_renewal_status_reason",
+				Description: "The reason that a renewal request was unsuccessful.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("RenewalSummary.RenewalStatusReason"),
```

Comment:
remove duplicate renewal?  i.e `renewal_summary_status_reason`

Reply:
Where's the duplicate? Do you mean `renewal_summary_renewal_status` and `renewal_summary_renewal_status_reason` ? They are different, first one is the actual renewal status, and the second one is *failiure* reason (if any, I guess). I'll rename it to `renewal_summary_failure_reason`.

Reply:
I meant the renaming of the following columns, sorry if it wasn't clear.